aries-site/src/pages/templates/popover.mdx

```diff
@@ -0,0 +1,69 @@
+import { Example } from '../../layouts';
+import {
+  PopoverAnatomy,
+  PopoverSimpleExample,
+  PopoverInlineExample,
+} from '../../examples';
+
+<Example
+  figma="https://www.figma.com/design/5uOrsL2qk8XwwH8C1ZpDb6/HPE-Design-System-Components?node-id=34141-2"
+  width="100%"
+  code='https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/templates/popover/PopoverSimpleExample.js'
+>
+  <PopoverSimpleExample />
+</Example>
+
+## Use cases
+### Providing inline contextual help
+
+A Popover can be triggered through an icon-only button to offer helpful contextual information in a non-intrusive manner.
+
+<Example
+  figma="https://www.figma.com/design/5uOrsL2qk8XwwH8C1ZpDb6/HPE-Design-System-Components?node-id=34141-2"
+  width="100%"
+  code='https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/templates/popover/PopoverInlineExample.js'
+>
+  <PopoverInlineExample />
+</Example>
+
+## Anatomy
+
+<Example caption="Diagram illustrating the 4 parts of a Popover." plain>
+  <PopoverAnatomy />
+</Example>
+
+| Label | Region        | Purpose                              | Required | Notes                                      |
+| :---: | ------------- | ------------------------------------ | :------: | ------------------------------------------ |
+| **1** | **Title**     | Identifies the Popover.               | Optional |                                            |
+| **2** | **Close Button** | Closes the Popover.                | Required |                                            |
+| **3** | **Body**      | Provides contextual information.      | Required | May include non-text elements.             |
+| **4** | **Footer**    | Contains additional actions.          | Optional | May include anchors.                       |
+
```

Comment:
Are we planning to provide spacing annotations as well? Similar to https://design-system.hpe.design/components/layer#anatomy

Reply:
![Screenshot 2024-10-16 at 9 48 37 PM](https://github.com/user-attachments/assets/e8963513-ac65-4bca-8426-dd182db8574d)
yes I have these then will add the table for when the screen is smaller is has the values

Reply:
beautiful.